src/js/components/TextInput/TextInput.js

```diff
@@ -232,6 +252,9 @@ const TextInput = forwardRef(
       placeholder && typeof placeholder !== 'string' && !value;
 
     let drop;
+    const extraProps = {
```

Comment:
Why do we need this variable, since it's only used once, it would be cleaner to just have it inline: `onSelect={handleTextSelect.current}`

Reply:
I had that initially, but this was added to keep the tests from updating.